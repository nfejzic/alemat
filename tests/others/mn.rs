use alemat::{elements::Num, MathMl, MathMlFormatter};

#[test]
fn number() {
    let out = MathMl::with_content(Num::from(1)).render(&mut MathMlFormatter);

    crate::snap_test!(out, name: "others_number");
}
