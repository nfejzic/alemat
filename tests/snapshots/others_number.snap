---
source: tests/others/mn.rs
expression: out
---
<math><mn>1</mn></math>
